packages/tradershub/src/features/cfd/screens/MT5TradeScreen/MT5TradeLink/MT5TradeLink.tsx

```diff
@@ -40,7 +41,7 @@ const MT5TradeLink: FC<TMT5TradeLinkProps> = ({ app = 'linux', platform, webtrad
     };
 
     return (
-        <div className='flex items-center justify-between border-t border-system-light-secondary-background px-800 py-1200'>
+        <div className='flex items-center justify-between border-t-100 border-solid border-system-light-secondary-background px-800 py-1200'>
```

Comment:
I don't think you need border solid here and fyi `border-75` is equivalent to  `border = 1px`
```suggestion
        <div className='flex items-center justify-between border-t-100 border-system-light-secondary-background px-800 py-1200'>
```